secure-mobile-push/smp-android-client/app/src/main/java/de/adorsys/android/securemobilepushtest/MainActivity.java

```diff
@@ -62,6 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        SmsTool.createSmsConfig("BEGIN-MESSAGE", "END-MESSAGE", "0900123456", "0900654321", "0900900900");
```

Comment:
Fixed in commit:https://github.com/adorsys/secure-key-storage/pull/44/commits/eb220e2d2982822d17e2173262b0ad7e0fb3e81c